### PR TITLE
Adding new React UI item in collections

### PR DESCRIPTION
**Pull request by officialrajdeepsingh**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

Adding new React UI items in collections. It contains a list of Morden UI libraries based on React.

### CODE REVIEW

File: collections/react-ui/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ - chakra-ui/chakra-ui
+ - primefaces/primeng
+ - rewindui/rewindui
+ - palantir/blueprint
+ - rsuite/rsuite
+ - Semantic-Org/Semantic-UI-React
+
+---
+
+A React UI library or React component library of a collection of components. You can use it, and ready for production.
```
Comment (ahpook) [inline collections/react-ui/index.md:33]: This wording could be improved.

```suggestion
A React UI library or collection of React components. React is an open source JavaScript framework for designing user interfaces.
```

**Review by ahpook — CHANGES_REQUESTED** [review]
The collection looks good, but please fix the description. Thanks for the contribution!

File: collections/react-ui/index.md (hunk 1 above)
Comment (officialrajdeepsingh) [reply collections/react-ui/index.md]: Yes, thanks for help

**Comment by ipc103** [comment]
@officialrajdeepsingh - one (hopefully) last request here - we require the image name to match the name of the collection. Mind renaming `react.png` to `react-ui.png` to get our CI passing?

**Review by Briancarl07 — COMMENTED** [review]
Nice

**Review by ipc103 — APPROVED** [review]
👍 ❤️ Thanks for contributing!

**Review by ahpook — APPROVED** [review]

**Pull request merged**